README.md

```diff
@@ -945,11 +959,20 @@ console.log(results);
 //   usage: { readUnits: 1 }
 // }
 
-// Fetch the next page of results
-await index.listPaginated({
+// Grab the final vector ID matching prefix 'doc1#' using the paginationToken returned by the previous call
```

Comment:
Nit: "Fetch" is a verb we use elsewhere. "Get" might also make sense given the http verb. I'd avoid "Grab"

Reply:
Ah okay thanks, I'll change to fetch!